tests/adiabatic_initial_conditions_chunk_3d/statistics

```diff
@@ -8,5 +8,5 @@
 # 8: Iterations for Stokes solver
 # 9: Velocity iterations in Stokes preconditioner
 # 10: Schur complement iterations in Stokes preconditioner
-0 0.000000000000e+00 0.000000000000e+00  64  2312  729 0 222 143  23 
+0 0.000000000000e+00 0.000000000000e+00  64  2312  729 0 222 348 227 
```

Comment:
Yes. And only now I realize that you were right with your concerns in #1421, we do more iterations than we thought (348-143=205, instead of 200 cheap iterations we should be making). The deal.II FGMRES solver has a "feature" that it always make one more iteration per restart than asked for. It does not count the first iteration, because we do not do a least squares solve for the residual. I am not sure if this is right or wrong, but it is an independent issue, so this PR is ready to merge.